crates/gitbutler-branch-actions/src/undo_commit.rs

```diff
@@ -55,7 +57,8 @@ struct UndoResult {
     ownership_update: Vec<OwnershipClaim>,
 }
 
-fn inner_undo_commit(
+#[instrument(level = tracing::Level::DEBUG, skip(repository))]
+fn rebase_all_but_last(
```

Comment:
would `rebase_excluding` be a better name?

Reply:
Yes, that's actually what it's doing. Renaming it.